tenable/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: Nessus
```

Comment:
If we're monitoring nessus, should we name the integration nessus? ex `integrations-core/nessus`

Reply:
Removed that comment. Was a holdover.